core/eolearn/tests/test_fs_utils.py

```diff
@@ -60,6 +60,30 @@ def test_s3_filesystem():
         assert isinstance(filesystem, S3FS)
         assert filesystem.aws_access_key_id == custom_config.aws_access_key_id
         assert filesystem.aws_secret_access_key == custom_config.aws_secret_access_key
+        assert filesystem.aws_session_token is None
+
+
+@mock_s3
+def test_s3_filesystem_with_session_token():
```

Comment:
Instead of making a copy of the `test_s3_filesystem` test function could you just parameterize it by adding a decorator in form of 

```Python
@pytest.mark.parametrize("aws_session_token", [None, "fake-session-token"])
```